simple/src/main/java/com/github/commonsrdf/simple/BlankNodeImpl.java

```diff
@@ -29,12 +27,11 @@
  */
 class BlankNodeImpl implements BlankNode {
 
-	private static AtomicLong bnodeCounter = new AtomicLong();
 	private final String id;
 	private final Optional<Graph> localScope;
 
 	public BlankNodeImpl() {
-		this(Optional.empty(), "b:" + bnodeCounter.incrementAndGet());
+		this(Optional.empty(), UUID.randomUUID().toString());
```

Comment:
There shouldn't be a need as now the identifier will be a UUID anyway. Why
do you need to make a UUID from a UUID?
On 5 Feb 2015 22:54, "Peter Ansell" <notifications@github.com> wrote:

> In simple/src/main/java/com/github/commonsrdf/simple/BlankNodeImpl.java
> <https://github.com/commons-rdf/commons-rdf/pull/61#discussion_r24208131>:
>
> >  	private final String id;
> >  	private final Optional<Graph> localScope;
> >
> >  	public BlankNodeImpl() {
> > -		this(Optional.empty(), "b:" + bnodeCounter.incrementAndGet());
> > +		this(Optional.empty(), UUID.randomUUID().toString());
>
> Could you add the "b:" as a prefix back on so that it triggers the branch
> in ntriplesString?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/commons-rdf/commons-rdf/pull/61/files#r24208131>.
>